Frontend/ui-customer/src/Screens/OnBoardingFlow/EnterMobileNumberScreen/Controller.purs

```diff
@@ -119,23 +119,22 @@ eval (MobileNumberEditTextAction (PrimaryEditTextController.TextChanged id value
                                                     if value=="5000500050" then true else false 
                                                         else true 
                                     Nothing -> true 
+    let newState = state { props = state.props { isValidMobileNumber = isValidMobileNumber
+                                            , btnActiveMobileNuber = if (length value == 10 && isValidMobileNumber) then true else false}
```

Comment:
@ashkriti can you correct the spelling of this variable "btnActiveMobileNuber" => "btnActiveMobileNumber"

Reply:
Done